src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const rentCar = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return rentCar - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return rentCar - SHORT_TERM_DISCOUNT;
+  }
+
+  return rentCar;
 }
```

Comment:
The function `calculateRentalCost` is well-named and describes its purpose effectively. However, the use of `else if` after a return statement is unnecessary and can be simplified.